charts/smartmetserver/templates/persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.pvc }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.smartmetserver.name }}-{{ .Values.pvc.name }}
+spec:
+  storageClassName: {{ .Values.pvc.storageClassName }}
+  accessModes:
+    - {{ .Values.pvc.accessModes }}
+  resources:
+    requests:
+      storage: {{ .Values.pvc.storage }}
+{{ end }}
```

Comment:
Testing with local cluster, PersistentVolumeClaim is stuck in Pending state as there are no PersistentVolumes created by the Chart. If you're using some PersistentVolume provisioner it should still be possible (and default) to create some local PersistentVolume from this chart to enable easy install.

Reply:
Can you specify what do you mean by local? Some sort of volume just for easy testing and local to the node (emptyDir/hostPath) or PersistentVolume's volume type called [local](https://kubernetes.io/docs/concepts/storage/volumes/#local)?

Reply:
It can be either, main point is that the Chart should be by default easily installable to any cluster. For example adding PersistentVolume like this to the Chart works, you can have a variable to disable creating this if you don't need it in your deployments:

```
apiVersion: v1
kind: PersistentVolume
metadata:
  name: {{ .Values.smartmetserver.name }}-volume
spec:
  storageClassName: {{ .Values.pvc.storageClassName }}
  capacity:
    storage: {{ .Values.pvc.storage }}
  accessModes:
    - {{ .Values.pvc.accessModes }}
  claimRef:
    namespace: {{ .Release.Namespace }}
    name: {{ .Values.smartmetserver.name }}-{{ .Values.pvc.name }}
  hostPath:
    path: "/"
```

I realised now that in theory by setting `pvc: ""` you already don't create the pvc and use emptyDir, but for me atleast it's not working.

Reply:
Added Peristenvolume using hostPath in commit https://github.com/fmidev/helm-charts/pull/17/commits/3e6d5c5c6b37966a692bb3f8258656aed7e54c10.
Configuring write access is left to the user as the image runs as non-root. Automatic write access would require adding an init container.